modules/apps/frontend-css/frontend-css-web/src/main/resources/META-INF/resources/portal/_editor_ckeditor.scss

```diff
@@ -1,3 +1,108 @@
+.html-editor.portlet {
+	background-color: $light;
+}
+
+.cke {
+	border: 1px solid $secondary-l3;
+	border-radius: 4px;
+
+	.cke_inner {
+		border-radius: 4px;
+
+		.cke_top {
+			background-color: $white;
+			border-bottom: 1px solid $secondary-l3;
+			//border-radius: 4px;
+
+			.cke_toolbar {
+				&:not(:first-child) {
+					margin-left: 4px;
+				}
+
+				.cke_toolgroup {
+					a.cke_button {
+						border: none;
+						border-radius: 4px;
+						margin: 0 2px;
+						padding: 7px 8px;
+
+						&:last-child:after, &.cke_button_disabled:hover:last-child:after {
+							border-right: 1px solid $secondary-l3;
+							height: 24px;
+							right: -7px;
+						}
+
+						&.cke_button_disabled {
+							color: $secondary-l1;
+						}
+
+						&.cke_button_on {
+							background-color: rgba($dark, 0.06);
+
+							&:last-child:after {
+								top: 0;
+							}
+						}
+
+						&:hover {
```

Comment:
probably we need to cover other states like focus:

```
&:hover,
&:focus
...
```

We should review all of them by focusing, press button, mark them as visited, etc

Reply:
I think you can't focus on ckeditor buttons, if you find out how PLEASE tell me